applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -1,29 +1,33 @@
 @import model.Page
 @import com.gu.identity.model.EmailNewsletter
 @import com.gu.identity.model.EmailNewsletters._
+@import services.newsletters.EmailSignupAgent
+@import services.newsletters.NewsletterResponse
 @import views.support.`package`.Seq2zipWithRowInfo
 
 @import common.LinkTo
 
+@import com.gu.identity.model.GroupedNewsletters
+@import services.newsletters.NewsletterResponse
 @(signupPage: Page)(implicit request: RequestHeader, context: model.ApplicationContext)
 
 @manageEmailsUrl = @{"https://manage.theguardian.com/email-prefs"}
 
-@emailListCategoryList(theme: String, emailListings: List[EmailNewsletter]) = {
+@emailListCategoryList(theme: String, emailListings: List[NewsletterResponse]) = {
     <div class="newsletters-category__heading">
         @theme
     </div>
     <div class="newsletter-card__wrapper">
         @emailListings.zipWithRowInfo.map { case (emailListing, row) =>
-            <div class="newsletter-card" data-component="newsletter-card @emailListing.identityName">
+            <div class="newsletter-card" data-component="newsletter-card @emailListing.id">
                 <div class="newsletter-card__content js-newsletter-content">
 
                     <div class="newsletter-card__name">
                         @emailListing.name
                     </div>
 
                     <div>
-                        @emailListing.teaser
+                        @emailListing.description
```

Comment:
are teaser and description equivalent?

Reply:
For the most part they are. I think they stem from much earlier work where the description had to be reduced in length sometimes but having worked on this and looked it over, the differences that there are are typically tiny thing like removing a couple of words. Making the two things equivalent was one of the pieces of work I wanted to do on the general data model

Reply:
Ok well I guess if all the current descriptions fit their usage here then we're ok.